proj.sh

```diff
@@ -6,7 +6,8 @@ function generate-data {
   echo 'Data generation started.'
   source .env && yarn --cwd $DATA_GENERATOR_PATH start $DATA_FILES_PATH "$@"
   /bin/cp -rf $DATA_GENERATOR_PATH/data/* neo4j/import
-  /bin/cp  -f $DATA_GENERATOR_PATH/data/hpaRna.json api/src/data/
+  /bin/cp  -f $DATA_GENERATOR_PATH/data/hpaRna.json api/src/data/ # this line should be removed after HPA data is included in the dataOverlay folder
```

Comment:
It would make more sense to me if this comment would be a checklist item in https://github.com/MetabolicAtlas/private-issues/issues/124 . 

My feeling is that to-dos and to-do style comments in code are incompatible with planning practices (including scrum). In line with this thinking, I had created an [issue to clean up to-dos](https://github.com/MetabolicAtlas/private-issues/issues/34).

Reply:
Sounds good! I have updated the issue description now.